buffer.go

```diff
@@ -269,10 +278,20 @@ func (buf *Buffer) configure(schema *Schema) {
 			if sortingColumns[sortingIndex].Descending() {
 				column = &reversedColumnBuffer{column}
 			}
-			buf.sorted[sortingIndex] = column
+			sortedPlaceholders[sortingIndex] = &sortedColumn{
+				buffer:      column,
+				schemaIndex: columnIndex,
```

Comment:
What is schemaIndex useful for?

Reply:
@achille-roussel It just records the position of the column in the schema. I mean, it’s useful if the user needs to know which schema column a sorting buffer corresponds to.

Reply:
@achille-roussel Also, I was asking the same thing. So currently, the code silently ignores any non-existent columns in the sorting configuration. Do you think we should let the user know if there is a non-existent column, or is the current behavior good?

Reply:
Let's remove it if we don't have a use for it.